OpenOversight/tests/test_utils.py

```diff
@@ -233,7 +233,7 @@ def test_save_image_to_s3_and_db_returns_existing_image(
     test_png_bytes_io.close = lambda: None
     firstUpload = save_image_to_s3_and_db(test_png_bytes_io, 1, 1)
     secondUpload = save_image_to_s3_and_db(test_png_bytes_io, 1, 1)
-    assert type(secondUpload) == Image
+    assert isinstance(secondUpload, Image)
```

Comment:
The only meaningful changes are the use of `isinstance` instead of `type`, which can be validated by this statement.